src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -163,8 +185,18 @@ internal abstract partial class DefinitionItem
             ImmutableDictionary<string, string> properties = null,
             bool displayIfNoReferences = true)
         {
-            return new NonNavigatingDefinitionItem(
-                tags, displayParts, originationParts, properties, displayIfNoReferences);
+            properties = properties ?? ImmutableDictionary<string, string>.Empty;
+            properties = properties.Add(NonNavigable, NonNavigable);
```

Comment:
:bulb: If the value doesn't matter and these are getting serialized over the wire, use `string.Empty` as the value to save space.